test/mocha/30-database.js

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {meters} = require('bedrock-meter');
+const {cleanDB} = require('./helpers.js');
+const bnid = require('bnid');
+
+describe.only('Meters Database Tests', function() {
```

Comment:
Is this `.only` intentional?

Reply:
No this should not be there. I will remove.